tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/Translation.spec.ts

```diff
@@ -0,0 +1,191 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
```

Comment:
There are quite a bit of usage of umbracoApi.dictionary. Those should be changed to umbracoApi.translation

Reply:
For me, I prefer to keep "umbracoApi.dictionary" for the Translation enpoint as the name of these enpoints in swagger is "Dictionary"

Reply:
That makes sense!